docs/Makefile

```diff
@@ -1,9 +1,10 @@
 DMD=dmd
 GENERATED=../generated
 DMD_MAN_PAGE=man/man1/dmd.1
+DIFFABLE=1
```

Comment:
Don't you want `DIFFABLE` to be off by default (i.e. normal documentation builds include timestamps, commit SHA1s, etc.)?

Reply:
Indeed

Reply:
Indeed